<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Monitor</class>
 <widget class="QMainWindow" name="Monitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenDDS Monitor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0" rowspan="2">
     <widget class="QTreeView" name="repoView"/>
    </item>
    <item row="3" column="1">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Repository</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <widget class="QPushButton" name="repoAddButton">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>40</y>
           <width>92</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Add</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+A</string>
         </property>
        </widget>
        <widget class="QPushButton" name="repoRemoveButton">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>90</y>
           <width>92</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Remove</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+R</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="quitButton">
        <property name="text">
         <string>Quit</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+Q</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Current Repository:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="repoSelection">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>repository to monitor</string>
        </property>
        <property name="statusTip">
         <string>Select a repository to monitor</string>
        </property>
        <property name="minimumContentsLength">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Monitor.qrc"/>
 </resources>
 <connections/>
</ui>
